rs/orchestrator/src/boundary_node.rs

```diff
@@ -154,38 +160,18 @@ impl BoundaryNodeManager {
             .as_ref()
             .ok_or_else(|| OrchestratorError::DomainNameMissingError(self.node_id))?;
 
-        // TODO: Should these values be settable via config?
-        let args = vec![
-            format!("--listen-https-port=443"),
-            format!("--tls-hostname={}", domain_name),
-            format!("--tls-cert-path=/var/lib/ic/data/ic-boundary-tls.crt"),
-            format!("--tls-pkey-path=/var/lib/ic/data/ic-boundary-tls.key"),
-            format!("--tls-acme-credentials-path=/var/lib/ic/data"),
-            format!("--http-client-timeout-connect=3s"),
-            format!("--registry-disable-replicator"),
-            format!("--registry-local-store-path=/var/lib/ic/data/ic_registry_local_store"),
-            format!("--obs-log-journald"),
-            format!("--obs-metrics-addr=[::]:9324"),
-            format!("--rate-limit-per-second-per-subnet=1000"),
-            format!("--bouncer-enable"),
-            format!("--bouncer-ratelimit=600"),
-            format!("--bouncer-burst-size=1200"),
-            format!("--bouncer-ban-time=5m"),
-            format!("--bouncer-max-buckets=30000"),
-            format!("--bouncer-bucket-ttl=1m"),
-            format!("--cache-size=1GB"),
-            format!("--cache-max-item-size=10MB"),
-            format!("--cache-ttl=1s"),
-            format!("--shed-system-ewma=0.9"),
-            format!("--shed-system-cpu=0.9"),
-            format!("--shed-system-memory=0.9"),
-        ];
+        let env = env_file_reader::read_file("/opt/ic/share/ic-boundary.env").map_err(|e| {
+            OrchestratorError::IoError("unable to read ic-boundary environment variables".into(), e)
+        })?;
+
+        let args = vec![format!("--tls-hostname={}", domain_name)];
```

Comment:
optional nit: it's possible to put variables directly into the format string:
`format!("--tls-hostname={domain_name}")`

Reply:
Yes, but I left it as-is how it was made by the consensus team. Maybe will fix.

Reply:
I'm indifferent as long as it's consistent. In the past I was against it because rust-analyzer didn't fully support implicit named arguments, but now it does so I'm fine with it.